setup.py

```diff
@@ -32,13 +32,6 @@
     disable_openmp = False
 
 
-##########################
-VERSION = "1.9.10.dev0"
-ISRELEASED = False
-__version__ = VERSION
-##########################
-
```

Comment:
IMHO the main value add is having something less error-prone than this; in the release wheels I have this set (properly) to 1.10.0 but I had to do that with a local modification that differs from the release assets, which is not good.